### DISCUSSION TITLE
lockfile maintenance on-demand runs prevented by schedule

### DISCUSSION THREAD

**Discussion by rbtcollins** [renovatebot/renovate#23966] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

When I click on 'lock file maintenance' to schedule an immediate run of that, on the [Dependency Dashboard](https://github.com/rbtcollins/fs_at/issues/25) the resulting run doesn't create the PR/branch.

When I do the same task but via the [renovate status dashboard](https://developer.mend.io/github/rbtcollins/fs_at) it is created.

Assuming there is nothing wrong with my repository renovate yaml file, this should be reproducable in any repository.

### Relevant debug logs

<details><summary>Failure Logs</summary>
```
DEBUG: Package not scheduled (branch="renovate/lock-file-maintenance")
DEBUG: Skipping branch creation as not within schedule (branch="renovate/lock-file-maintenance")
```
</details>

<details><summary>Success Logs</summary>
```
DEBUG: Package not scheduled (branch="renovate/lock-file-maintenance")
DEBUG: Manual create/rebase requested via checkedBranches (branch="renovate/lock-file-maintenance")

```
</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Can you extract the full branch logs for the failure case? i.e. all debug messages matching `(branch="renovate/lock-file-maintenance") `
